pandas/core/dtypes/cast.py

```diff
@@ -731,47 +728,19 @@ def astype_nansafe(arr, dtype, copy=True, skipna=False):
     return arr.view(dtype)
 
 
-def maybe_convert_objects(
-    values, convert_dates=True, convert_numeric=True, convert_timedeltas=True, copy=True
-):
+def maybe_convert_objects(values: np.ndarray, convert_numeric: bool = True):
     """ if we have an object dtype, try to coerce dates and/or numbers """
+    validate_bool_kwarg(convert_numeric, "convert_numeric")
```

Comment:
can you add Parameters / Returns to the doc-string

Reply:
sure